src/pages/Signin/index.jsx

```diff
@@ -50,11 +52,11 @@ export class Signin extends React.Component {
             <div className="innerHalfContent">
               <div className="socialDiv">
                 Continue with
-                <FaceBook />
+                <FaceBookButton />
                 {' '}
-                <Twitter />
+
                 {' '}
-                <Google />
+                <GoogleButton responseGoogle={GoogleHelper} />
```

Comment:
Is there a reason why you extracted the function to a separate file and not declare it within this file?

Reply:
@adesege the function has been removed so that it is reusable and used as props. This is to avoid duplication.